scripts/src/main/java/com/mathewsachin/fategrandautomata/scripts/entrypoints/AutoBattle.kt

```diff
@@ -188,17 +189,7 @@ open class AutoBattle(
             }
         }
 
-        // Post-battle story is sometimes there.
-        if (prefs.storySkip) {
-            if (game.menuStorySkipRegion.exists(images.storySkip)) {
-                game.menuStorySkipClick.click()
-                0.5.seconds.wait()
-                game.menuStorySkipYesClick.click()
-            }
-        }
-
-        10.seconds.wait()
-
+        // TODO: Move to outer loop
         // Quest Completion reward. Exits the screen when it is presented.
         if (game.resultCeRewardRegion.exists(images.bond10Reward)) {
```

Comment:
Hold on, why is Bond 10 handled twice? One time above, one time below?

Reply:
IIRC, this was for the CE rewards after battle

Reply:
I see, so bond10Reward is used for both. But this code won't work anymore anyways because of timing issues.

Reply:
yeah, that is why I added that TODO there

Reply:
I don't think it would have worked reliably before